### PR TITLE
Update distillation job with sdk example

### PR DESCRIPTION
**Pull request by sanchez-alex**
# Description

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by kingernupur** [comment]
@sanchez-alex Have you added workflows for these new notebooks?

**Comment by sanchez-alex** [comment]
> @sanchez-alex Have you added workflows for these new notebooks?
One of these notebooks has a workflow already, the rest of them do not have a workflow to avoid too many inference requests to the MaaS endpoint

**Review by hakotesova — DISMISSED** [review]

**Comment by jayesh-tanna** [comment]
Consider to add team ownership entry into notebook owning file.

File: cli/foundation-models/system/distillation/conversation/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+2. If the `ml` extension is not installed, run `az extension add -n ml`
+
+Run the Distillation CLI command pointing to the .YAML file in this folder and fill out the Azure ML IDs needed:
+
+```text
+az ml job create --file distillation_conversation.yaml --workspace-name [YOUR_AZURE_WORKSPACE] --resource-group [YOUR_AZURE_RESOURCE_GROUP] --subscription [YOUR_AZURE_SUBSCRIPTION]
+```
+
+## 2. Deploy to Endpoint
+Once the job finishes running, fill out the serverless_endpoint.yaml file in this folder. The necessary information can be found by
```
Comment (msp8955) [inline cli/foundation-models/system/distillation/conversation/README.md:15]: Once distilled model is ready*

File: cli/foundation-models/system/distillation/conversation/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Run the Distillation CLI command pointing to the .YAML file in this folder and fill out the Azure ML IDs needed:
+
+```text
+az ml job create --file distillation_conversation.yaml --workspace-name [YOUR_AZURE_WORKSPACE] --resource-group [YOUR_AZURE_RESOURCE_GROUP] --subscription [YOUR_AZURE_SUBSCRIPTION]
+```
+
+## 2. Deploy to Endpoint
+Once the job finishes running, fill out the serverless_endpoint.yaml file in this folder. The necessary information can be found by 
+1. Navigating to the `model` tab in [ml studio](https://ml.azure.com).
```
Comment (msp8955) [inline cli/foundation-models/system/distillation/conversation/README.md:16]: also we should be able to deploy using UI as well through ml studio or ai studio

File: cli/foundation-models/system/distillation/conversation/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Distillation with CLI (Conversation)
+
+## 1. Create the Job
```
Comment (msp8955) [inline cli/foundation-models/system/distillation/conversation/README.md:3]: Can you have a link for SDK notebook for the .jsonl creation reference?

**Review by msp8955 — DISMISSED** [review]

**Comment by sanchez-alex** [comment]
> Consider to add team ownership entry into notebook owning file.

Ownership entry in notebook owning file already there

**Comment by msp8955** [comment]
Can you also add a link to cli examples in the main README? https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/distillation/README.md

File: cli/foundation-models/system/distillation/conversation/README.md (hunk 1 above)
Comment (sanchez-alex) [reply cli/foundation-models/system/distillation/conversation/README.md]: Done

File: cli/foundation-models/system/distillation/conversation/README.md (hunk 2 above)
Comment (sanchez-alex) [reply cli/foundation-models/system/distillation/conversation/README.md]: Added section giving additional info on UI deployment vs CLI deployment

File: cli/foundation-models/system/distillation/conversation/README.md (hunk 3 above)
Comment (sanchez-alex) [reply cli/foundation-models/system/distillation/conversation/README.md:3]: Added note beneath this section to see the notebook example for the reference

**Review by jayesh-tanna — APPROVED** [review]

**Pull request merged**